mapie/regression/time_series_regression.py

```diff
@@ -0,0 +1,343 @@
+from __future__ import annotations
```

Comment:
Difficult to see with the github diff, but have you changed any of the code from the previous code to this one?